src/en/tut-google.md

```diff
@@ -115,19 +98,19 @@ into environments that can be seamlessly managed, deployed and scaled.
 
 For example, different models can be deployed to different regions. You can see
 which regions your cloud supports with the `juju show-cloud google` command,
-and create a new model hosted on `europe-west1` with the following:
+and create a new model hosted on `us-east1` with the following:
 
 ```bash
-juju add-model gce-test europe-west1
+juju add-model default us-east1
```

Comment:
Will this actually work? Isn't there already a model called 'default'? 
I believe @degville added this to slowly introduce the concept of creating extra models

Reply:
I was doing more adding and removing of models due to things not working. I will change it to 'gce-test'.

Reply:
:+1: